Description: Fixes spelling errors.
Author: Marcio de Souza Oliveira <m.desouza20@gmail.com>
Last-Update: 2017-07-27
Index: nload-0.7.4/docs/nload.1.in
===================================================================
--- nload-0.7.4.orig/docs/nload.1.in
+++ nload-0.7.4/docs/nload.1.in
@@ -29,7 +29,7 @@ nload \- displays the current network us
 .B nload
 is a console application which monitors network traffic and bandwidth usage in real time.
 It visualizes the in- and outgoing traffic using two graphs and provides additional info
-like the total amount of transfered data and min/max network usage.
+like the total amount of transferred data and min/max network usage.
 
 .SH USAGE
 When running
